src/components/card/index.tsx

```diff
@@ -1,166 +1,218 @@
-          >
-            {message?.elements?.title || 'Title Here'}
-          </text>
-          <text
-            className="titleText"
-            style={{ ...defaultTextStyles, ...textStyle }}
-          >
-            {message?.elements?.body}
-          </text>
-        </div>
-        <div style={cardButtons}>
-          {message?.elements?.buttons?.map((button: any, index: number) => (
-            <button
-              key={index}
-              disabled={index === 0 ? disablePrimaryBtn : disableSecondaryBtn}
-              style={
-                index === 0
-                  ? disablePrimaryBtn
-                    ? { ...defaultButtonStyles, ...primaryDisableBtnStyle }
-                    : { ...defaultButtonStyles, ...primaryBtnStyle }
-                  : disableSecondaryBtn
-                  ? { ...defaultButtonStyles, ...secondaryDisableBtnStyle }
-                  : { ...defaultButtonStyles, ...secondaryBtnStyle }
-              }
-              onClick={() => {
-                const clickedUrl =
-                  button?.action?.data?.trim() || button?.action?.type || '';
-                embeddedManager.handleEmbeddedClick(
-                  message,
-                  button?.id,
-                  clickedUrl
-                );
-                embeddedManager.trackEmbeddedClick(
-                  message,
-                  button?.id,
-                  clickedUrl
-                );
-              }}
-            >
-              {button.title ? button.title : `Button ${index + 1}`}
-            </button>
-          ))}
-        </div>
+  return `
+    <style>${mediaStyle}</style>
+    <div 
+      class="card"
+      id="${parentId}"
+      name="${message?.metadata?.messageId}-card"
+      style="${defaultCardStyles}; ${parentStyle || ''}" 
+    >
+      ${
+        message?.elements?.mediaUrl
+          ? `<img class="card" id="${imageId}" style="${defaultImageStyles}; ${
+              imgStyle || ''
+            }" 
+          src="${message?.elements?.mediaUrl}"/>`
+          : ''
+      }
+      <div class="card" style="${defaultTextParentStyles}">
+        <text class="titleText card"  id="${titleId}" style="${defaultTitleStyles}; ${
+    titleStyle || ''
+  }">
+          ${message?.elements?.title || 'Title Here'}
+        </text>
+        <text class="titleText card" id="${textId}" style="${defaultTextStyles}; ${
+    textStyle || ''
+  }">
+          ${message?.elements?.body}
+        </text>
+      </div>
+      <div class="card" style="${cardButtons}">
+        ${message?.elements?.buttons
+          ?.map((button: any, index: number) => {
+            const buttonStyle =
+              index === 0 ? primaryBtnStyle : secondaryBtnStyle;
+            const disableStyle =
+              index === 0 ? primaryDisableBtnStyle : secondaryDisableBtnStyle;
+            return `
+              <button 
+                key="${index}" 
+                ${
+                  index === 0
```

Comment:
same comment about the button styles as above

Reply:
Done